core/src/main/java/io/grpc/internal/AutoConfiguredLoadBalancerFactory.java

```diff
@@ -204,18 +213,18 @@ LoadBalancerProvider getDelegateProvider() {
      *   <li>User provided lb on the channel.  This is a degenerate case and not handled here.
      *       This options is deprecated.</li>
      *   <li>Policy from "loadBalancingConfig" if present.  This is not covered here.</li>
-     *   <li>"grpclb" if any gRPC LB balancer addresses are present</li>
      *   <li>The policy from "loadBalancingPolicy" if present</li>
+     *   <li>"grpclb" if any gRPC LB balancer addresses are present</li>
      *   <li>"pick_first" if the service config choice does not specify</li>
      * </ol>
      *
      * @param servers The list of servers reported
-     * @param config the service config object
+     * @param mcsc the parsed {@link ManagedChannelServiceConfig}
```

Comment:
This method only uses `mcsc.getLoadBalancingConfig()` which is a `PolicySelection`. We should only pass it to this method.

This method is focused on handling the GRPCLB special case, the whole javadoc needs to be rewritten to reflect that. The terminology is currently around the raw config, and that needs to be changed too.